.storybook/iframe.html

```diff
@@ -9,6 +9,7 @@
       }
     </script>
     <title>React Storybook</title>
+    <link type="text/css" rel="stylesheet" href="//fast.fonts.net/cssapi/f7f47a40-b25b-44ee-9f9c-cfdfc8bb2741.css">
```

Comment:
This may not work in the deployed storybook b/c of the allowed domains for the fonts.